src/routing/kad-dht.md

```diff
@@ -0,0 +1,666 @@
+---
+title: Kademlia DHT
+description: >
+  The IPFS Distributed Hash Table (DHT) specification defines a structured
+  overlay network used for peer routing and content routing in the
+  InterPlanetary File System (IPFS). It extends the libp2p Kademlia DHT
+  specification, adapting and adding features to support IPFS-specific
+  requirements.
+date: 2025-03-18
+maturity: reliable
```

Comment:
Set to `reliable` since this spec transcribes the network behavior as it currently is